### ISSUE TITLE
When a user is registering, have them enter in their first name and last name

### ISSUE THREAD

**Issue by Redmayspecial** [unfoldingWord-dev/translationDatabaseWeb#488]
## User Story
As a manager, I would like to request that a user enter in their name, not a username/alias, in order to identify themselves when they register.

**Comment by vleong2332** [comment]
Do we actually want to get rid of the username?

**Comment by markalolo** [comment]
No, we want to add collecting their real name and also email address if we are not already doing that.
